android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java

```diff
@@ -109,10 +106,19 @@ public JitsiMeetView(Context context, AttributeSet attrs, int defStyle) {
         initialize(context);
     }
 
-    @Override
+    /**
+     * Releases the React resources (specifically the {@link ReactRootView})
+     * associated with this view.
+     *
+     * MUST be called when the {@link Activity} holding this view is destroyed,
+     * typically in the {@code onDestroy} method.
+     */
     public void dispose() {
```

Comment:
should we make this private?

Reply:
No, this is necessary for those integrating the view in their own activity, we do support that.